### ISSUE TITLE
Migration cannot be reversed

### ISSUE THREAD

**Issue by benjaoming** [torchbox/wagtail#2072]
I'm trying to rerun migrations. This is a nice way of re-creating tables. Ideally, I'm thinking about running `python manage.py migrate wagtailcore zero`.

However, this migration is quite heavy:

```
  Unapplying wagtailimages.0004_make_focal_point_key_not_nullable...Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "local/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "local/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "local/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
    output = self.handle(*args, **options)
  File "local/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 161, in handle
    executor.migrate(targets, plan, fake=options.get("fake", False))
  File "local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 70, in migrate
    self.unapply_migration(migration, fake=fake)
  File "local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 122, in unapply_migration
    migration.unapply(project_state, schema_editor)
  File "local/lib/python2.7/site-packages/django/db/migrations/migration.py", line 133, in unapply
    raise Migration.IrreversibleError("Operation %s in %s is not reversible" % (operation, self))
django.db.migrations.migration.IrreversibleError: Operation <RunPython <function remove_duplicate_renditions at 0x7f2f5645cc80>> in wagtailimages.0004_make_focal_point_key_not_nullable is not reversible
```

But it looks like it "just" removes duplicates from the DB. So it should be safe to have a no-op as reverse function.

**Comment by benjaoming** [comment]
The above is just an example, but there were a couple of other cases of `RunPython` in the codebase that now have reverse operations in the PR.

**Comment by kaedroho** [comment]
Fixed by #2073
